### PR TITLE
Update Clarity and its peer dependencies

### PR DESCRIPTION
**Pull request by johnstill**

### CODE REVIEW

**Comment by calbach** [comment]
Added you to "snyk", but the complaint is:

Cross-Site Scripting (XSS)
Vulnerable module: bootstrap
Introduced through: bootstrap@4.0.0-alpha.5
Detailed paths
Introduced through: aou-ui-angular@all-of-us/workbench#24b19af3947113f0a7f5e5d99164d67a893fd32f › bootstrap@4.0.0-alpha.5
Remediation: Upgrade to bootstrap@4.0.0.

File: ui/package.json [hunk 1]
```diff
@@ -30,15 +30,15 @@
...
+    "@clr/angular": "^0.11.12",
+    "@clr/icons": "^0.11.12",
+    "@clr/ui": "^0.11.12",
     "@swimlane/ngx-charts": "^7.1.1",
     "@types/geojson": "^1.0.6",
     "@webcomponents/custom-elements": "^1.0.0",
     "angular-2-local-storage": "^1.0.1",
     "angular2-highcharts": "^0.5.5",
-    "bootstrap": "^4.0.0-alpha.5",
+    "bootstrap": "4.0.0-alpha.5",
```
Comment (calbach) [inline ui/package.json]: Per Snyk, please change this 4.0.0 or higher

Comment (johnstill) [reply ui/package.json]: It'll probably work with bootstrap 4, but the reason we have this is [Clarity UI requires _exactly_ this version](https://github.com/vmware/clarity/blob/ae0e115601472bbb9abdf956d19d4a9739ea698d/npm/clr-ui/package.json#L12)

Comment (johnstill) [reply ui/package.json]: I opened an issue with Clarity b/c it seems weird to me that they should be pinned to exactly an alpha version.  This smells a lot like "no-one bothered updating our packaging data when bootstrap moved out of alpha."

Comment (calbach) [reply ui/package.json]: Luckily it would seem some good citizen already filed an issue against them: https://github.com/vmware/clarity/issues/2169

**Comment by johnstill** [comment]
Ok so - about the bootstrap v4 vs v4.0.0-alpha.5 thing - apparently there _are_ some breaking changes to the way the grid works which is why clarity is pinned at alpha.5 (confer the issue referenced in this PR's comments for more info).  Our test battery runs and the site seems to work fine with bootstrap v4, so now the question - do we ignore Clarity's peerDependency or do we ignore Snyk's warning against v4.0.0-alpha.5?  For the latter, I don't know about the security implications.  For the former, I _think_ the worst case scenario is that we can't use some features of the grid system like pull and push classes.  But we're note using the grid system much anyways.

Thoughts?

**Comment by calbach** [comment]
Given those options, I'd probably go with 4.0.0. If we wind up needing the grid stuff, hopefully they'll be off the insecure version by then.

**Comment by johnstill** [comment]
Ok, merging.  For now we make Clarity unhappy rather than Snyk.

**Pull request merged**